src/gh_pr/utils/notifications.py

```
@@ -0,0 +1,378 @@
+            True if successful
+        """
+        try:
+            # Build AppleScript with proper quoting
+            script = f'display notification {self._as_quote(message)} with title {self._as_quote(title)}'
+            if subtitle:
+                script += f' subtitle {self._as_quote(subtitle)}'
+
+            if self.config.sound:
+                script += ' sound name "default"'
+
+            result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)